src/TeamSpeak3QueryApi/Specialized/TeamSpeakClient.cs

```diff
@@ -230,6 +232,17 @@ public Task<IReadOnlyList<ClientBan>> BanClient(GetClientInfo client, TimeSpan d
             return DataProxy.SerializeGeneric<GetClientInfo>(res);
         }
 
+        public async Task<GetClientDetailedInfo> GetClientInfo(GetClientInfo client) => await GetClientInfo(client.Id);
+
```

Comment:
We could just return `GetClientInfo(...)` and make the method non-`async`. Sorry if I overlooked this at first.